web/index.js

```diff
@@ -18,10 +18,11 @@ var options = {
   redisUrl: process.env.REDIS_URL
 };
 
-var server = require('./server')(options);
+const start = async () => {
```

Comment:
made it a function since `await` can only be used inside async functions and not inside the global scope